### PR TITLE
LPD-31640 Fix UPS CX

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/LPD-31640

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

File: workspaces/liferay-ups-workspace/client-extensions/liferay-ups-commerce-shipping-engine/src/main/resources/application-default.properties [hunk 1]
```diff
@@ -1,3 +1,11 @@
+#
+# LXC
+#
+
+com.liferay.lxc.dxp.domains=localhost:8080
+com.liferay.lxc.dxp.mainDomain=localhost:8080
+com.liferay.lxc.dxp.server.protocol=http
```
Comment (luca-pellizzon) [inline workspaces/liferay-ups-workspace/client-extensions/liferay-ups-commerce-shipping-engine/src/main/resources/application-default.properties:7]: these lines should not be sent to upstream, please remove them

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]
just a minor thing, everything else looks good

File: workspaces/liferay-ups-workspace/client-extensions/liferay-ups-commerce-shipping-engine/src/main/resources/application-default.properties (hunk 1 above)
Comment (alessiorendina) [reply workspaces/liferay-ups-workspace/client-extensions/liferay-ups-commerce-shipping-engine/src/main/resources/application-default.properties:7]: I saw there are in master under a couple of CX like Adyen, Stripe, etc.

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:stop

**Comment by alessiorendina** [comment]
ci:forward

**Pull request closed without merging**